instrumentation/jms-1.1/src/main/java/io/opentelemetry/instrumentation/auto/jms/JMSDecorator.java

```diff
@@ -38,20 +38,20 @@
   public static final Tracer TRACER = OpenTelemetry.getTracer("io.opentelemetry.auto.jms-1.1");
 
   public String spanNameForReceive(Message message) {
```

Comment:
Do we need both this method and `spanNameForConsumer` below?

Reply:
removed